packages/ui-extensions/docs/surfaces/checkout/reference/apis/storage.doc.ts

```diff
@@ -24,6 +25,7 @@ const data: ReferenceEntityTemplateSchema = {
       type: 'UseStorageGeneratedType',
     },
   ],
+  defaultExample: getExample('storage', ['jsx']),
```

Comment:
- [ ] @lucasuyezu made these changes
- [ ] @lucasuyezu approves these changes for the `2024-10` stable release.

commit: https://github.com/Shopify/checkout-web/commit/4442630fa51eb5ff0a4f32c7edebdcac38fcc765